app/views/content/non-uk-teachers/non-uk-qualifications.md

```diff
@@ -72,6 +71,7 @@ To train to teach in England, you will need to have:
 
 A GCSE is an academic qualification awarded for exams in England, usually taken at the age of 16.
 
+$traininuk$
```

Comment:
The h2 below this has gone slightly screwy?
![image](https://github.com/user-attachments/assets/c91d8568-725a-4a4e-88cb-2a32547e1bfd)